src/vdaf/hits.rs

```diff
@@ -850,14 +848,16 @@ mod tests {
         let mut round2: Vec<HitsPrepareMessage<I::Field>> = Vec::with_capacity(2);
         let mut state2: Vec<HitsPrepareStep<I::Field>> = Vec::with_capacity(2);
         for state in state1.into_iter() {
-            let (state, msg) = hits.prepare_next(state, round1.clone())?;
+            let (state, msg) =
+                hits.prepare_next(state, hits.prepare_preprocess(round1.clone())?)?;
```

Comment:
Is the `clone()` still needed or would a mutable reference to `round1` suffice?

Reply:
No because `prepare_preprocess` takes in an `IntoIterator`.